fastly/resource_fastly_alert_test.go

```diff
@@ -193,6 +193,69 @@ func TestAccFastlyAlert_basic_stats_aggregate(t *testing.T) {
 	})
 }
 
+func TestAccFastlyAlert_basic_stats_aggregate_percent(t *testing.T) {
```

Comment:
We should really change all these test functions to remove the underscores...

e.g.
```suggestion
func TestAccFastlyAlertBasicStatsAggregatePercent(t *testing.T) {
```

Reply:
I changed four of them to removed the `_`